src/components/ContributionsEpic.tsx

```diff
@@ -226,6 +228,34 @@ export const ContributionsEpic: React.FC<Props> = ({
                     className={paymentImageStyles}
                 />
             </div>
+
+            {showReminderFields && (
+                <ContributionsEpicReminder
+                    reminderDate={showReminderFields.reminderDate}
+                    reminderDateAsString={showReminderFields.reminderDateAsString}
+                />
+            )}
         </section>
     );
 };
+
+export interface SlotComponent {
+    component: JSX.Element;
+    js: string;
+}
+
+export const contributionsEpicSlot = (props: Props): SlotComponent => {
+    let js = '';
+    if (props.variant.showReminderFields) {
+        const contributionsReminderUrl =
+            process.env.NODE_ENV === 'production'
+                ? 'https://contribution-reminders.support.guardianapis.com/remind-me'
+                : 'https://contribution-reminders-code.support.guardianapis.com/remind-me';
+        const initScript = componentJs.toString();
+        js = initScript.replace(/%%CONTRIBUTIONS_REMINDER_URL%%/g, contributionsReminderUrl);
+    }
+    return {
+        component: <ContributionsEpic {...props} />,
```

Comment:
I've got a feeling I might be misremembering, maybe it's a Flow thing, but do we lose out on some type checking stuff here by using a spread instead of passing explicitly?

Reply:
I've checked and when I'm building that `props` object in `server.tsx` I do get the expected type checking.